bpfman/src/bpf.rs

```diff
@@ -859,8 +873,14 @@ impl BpfManager {
 
         // Call Aya to get ALL the loaded eBPF programs, and loop through each one.
         loaded_programs()
-            .map(|p| {
-                let prog = p.map_err(BpfmanError::BpfProgramError)?;
+            .filter_map(|p| match p {
```

Comment:
```suggestion
            .flat_map(|p| p.ok())
```
Which will silently ignore errors - there is no point in logging warn messages since there is nothing the user can do about it. This also avoids using option types in the map closure that immediately follows this.

Reply:
Done!